### PR TITLE
Fix bug checkboxes select all constantly visible

### PR DESCRIPTION
**Pull request by miqur**
Checkboxes and selected all options are constantly visible now.

### CODE REVIEW

File: css/interface.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
+/* For Safari to hide scrollbar */
+html {
+  overflow:   scroll;
```
Comment (squallstar) [inline css/interface.css]: only two spaces after `:`

File: js/interface.js [hunk 2]
```diff
@@ -374,6 +380,10 @@ function addFile(file) {
   currentFiles.push(file);
   $folderContents.append(templates.file(file));
   $('.empty-state').removeClass('active');
+  $('.new-menu').removeClass('active');
+  // Toggle checkbox header to false
```
Comment (squallstar) [inline js/interface.js]: empty line before comments

File: js/interface.js [hunk 3]
```diff
@@ -746,11 +756,17 @@ $('.file-manager-wrapper')
           Fliplet.Media.Folders.delete($element.attr('data-id')).then(function() {
             $element.remove();
             checkboxStatus();
+            // Toggle checkbox header to false
```
Comment (squallstar) [inline js/interface.js]: empty line before comments

File: js/interface.js [hunk 4]
```diff
@@ -746,11 +756,17 @@ $('.file-manager-wrapper')
...
+            $('.file-table-header input[type="checkbox"]').prop('checked', false);
+            $('.file-cell.selectable').css({'opacity': '0', 'visibility': 'hidden'});
           });
         } else {
           Fliplet.Media.Files.delete($element.attr('data-id')).then(function() {
             $element.remove();
             checkboxStatus();
+            // Toggle checkbox header to false
```
Comment (squallstar) [inline js/interface.js]: empty line before comments

**Comment by squallstar** [comment]
@miqur in the PR description could you add a line as follows:

> ref https://github.com/Fliplet/fliplet-studio/issues/123

Where `123` is the original ticket ID we've given? It should be in your Trello board. Thansk

**Comment by tonytlwu** [comment]
For clarity, this issue is not related to https://github.com/Fliplet/fliplet-studio/issues/123 (GitHub has picked up on the relation).

**Comment by miqur** [comment]
> @miqur in the PR description could you add a line as follows:
> 
> > ref [Fliplet/fliplet-studio#123](https://github.com/Fliplet/fliplet-studio/issues/123)
> 
> Where `123` is the original ticket ID we've given? It should be in your Trello board. Thanks

**Comment by miqur** [comment]
Closed by mistake

File: js/interface.js (hunk 2 above)
Comment (miqur) [reply js/interface.js]: fixed

File: js/interface.js (hunk 3 above)
Comment (miqur) [reply js/interface.js]: fixed

Comment (miqur) [reply js/interface.js]: fixed

File: js/interface.js (hunk 4 above)
Comment (miqur) [reply js/interface.js]: fixed

**Comment by squallstar** [comment]
@miqur Olga should be able to send you the ticket ID.

File: css/interface.css [hunk 5]
```diff
@@ -1,3 +1,11 @@
+/* For Safari to hide scrollbar */
+html {
+  overflow: scroll;
+}
+::-webkit-scrollbar {
+  width: 0;
+  background: transparent; /* make scrollbar transparent */
```
Comment (tonytlwu) [inline css/interface.css]: Why are we hiding scrollbars?

Comment (miqur) [reply css/interface.css]: In the Safari when we checked all items they are moving to the left for width of scrollbar.

Comment (tonytlwu) [reply css/interface.css]: Can you demonstrate how this looks and behaves before and after your change?

Comment (miqur) [reply css/interface.css]: Before changes.
with scrollbar
After changes.
without scrollbar

Comment (tonytlwu) [reply css/interface.css]: Thanks. I wanted to make sure we were hiding the scrollbar when the scrollbar is necessary.

This looks good.

**Pull request merged**